### PR TITLE
Updated gauge slider math and start of gear selector changes

### PR DESCRIPTION
**Pull request by dustinsterk**
Updated gauge slider math and start of gear selector changes

### CODE REVIEW

File: LFA.qml [hunk 1]
```diff
@@ -122,6 +122,17 @@ Item {
 
     ////////// TRANSAXLE GEAR VARIABLES //////////////////////////////////////
     property real gearpos: rpmtest.geardata
+    property string gearinfo:    if(gearpos===9)"P"
```
Comment (DavidThomasStewart) [inline LFA.qml:125]: I'd replace the multiple if's with this:

 property string gearinfo: switch (gearpos) {
        case 0: return "N";
        case 1: return "1";
        case 2: return "2";
        case 3: return "3";
        case 4: return "4";
        case 5: return "5";
        case 6: return "6";
        case 7: return "7";
        case 8: return "8";
        case 9: return "P";
        case 10: return "R";
        default: return ""; // 100 is the value that says do not display gear position
    }

File: LFA.qml [hunk 2]
```diff
@@ -676,7 +687,12 @@ Item {
 
         Rectangle {
             x: -135
-            y: 205 - ((root.watertempf - 31) * 0.70192)
```
Comment (DavidThomasStewart) [inline LFA.qml:679]: Thank you for this. The math hurt my soul.

File: LFA.qml [hunk 3]
```diff
@@ -876,7 +892,12 @@ Item {
 
         Rectangle {
             x: 491
-            y: 205 - ((root.oiltempf - 31) * 0.70192)
```
Comment (DavidThomasStewart) [inline LFA.qml:879]: Also here.

**Review by DavidThomasStewart — CHANGES_REQUESTED** [review]
Just change the multiple if's with the switch and we're golden.

File: LFA.qml (hunk 3 above)
Comment (dustinsterk) [reply LFA.qml:879]: Anytime!

File: LFA.qml (hunk 2 above)
Comment (dustinsterk) [reply LFA.qml:679]: :) Haha...yeah took me a few to get right.

**Review by dustinsterk — COMMENTED** [review]
Updated.

**Review by DavidThomasStewart — APPROVED** [review]
Booyah!

**Pull request merged**